client/src/services/payment.service.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from 'react-query'
+import { request } from 'utilities/axios-utils'
+
+const url = '/orders'
```

Comment:
Duplicate for specific actions. For example:

const API_ADD_ODER = '/orders'
const API_UPDATE_PAYMENT_STATUS = '/orders/{orderId}'

with API_UPDATE_PAYMENT_STATUS, we can use this logic to make it looks cool 

`export const updatePaymentStatus = (order: order) => {
    return request({
        url: API_UPDATE_PAYMENT_STATUS .replace('{orderId}', order.id),
        method: 'PATCH',
        data: {
            paymentStatus: order.paymentStatus
        }
    })
}`

with the method of the request let's make a constant:
const REQUEST_METHOD = { PATCH: "PATCH", ...} 

and replace the whole project